application/client/src/app/ui/elements/timezones/timezone.ts

```diff
@@ -1,17 +1,21 @@
 import * as moment_timezone from 'moment-timezone';
-import * as regex from '@platform/env/regex';
+import { Matcher } from '@wasm/matcher/pkg/matcher';
 
 export class Timezone {
     public readonly name: string;
     public readonly utc: string;
     public readonly offset: number;
     public hidden: boolean = false;
 
-    private _filter: string = '';
+    private _matcher: Matcher = Matcher.new();
+    private _html_name: string;
+    private _html_utc: string;
```

Comment:
In JS name convention is htmlName instead html_name
We have probably one exception - when we have deal with parsing income from rust-world JSON... but at places where we can keep JS naming convention, we should keep it